pipeline_steps/github.groovy

```diff
@@ -70,12 +70,33 @@ void add_issue_url_to_pr(){
   return null
 }
 
-
-void add_comment_to_pr(String body){
-  List org_repo = env.ghprbGhRepository.split("/")
+/**
+ * If called with add_comment_to_pr("comment"), then comment will be added to
+ * the PR that triggered this job (using ghprb env vars to discover the pr).
+ *
+ * To add comments to an alternate pr, supply repo in the form "org/repo" and
+ * prid as "id" where id is the pull request number.
+ */
+void add_comment_to_pr(String body,
+                       String repo_url="use_env",
+                       String prid="use_env",
+                       Boolean allowDuplicates=false){
+  if ( repo_url == "use_env"){
+    repo_url = env.ghprbGhRepository
+  }
+  if (prid == "use_env"){
+    prid = env.ghprbPullId
+  }
```

Comment:
Having them independently able to use the ghprb values allows for bugs, for example if only the repo URL was overridden on a pull request job, what would that mean? Pull request IDs are not shared so it is almost certain to reference the wrong pull request.